js/content.js

```diff
@@ -0,0 +1,46 @@
+const bridgeEvents = {
+    CONNECT: 'loadster_connect_extension',
+    CONNECTED: 'loadster_connected_extension',
+    DISCONNECTED: 'loadster_disconnected_extension',
+    SEND: 'loadster_post_message',
+    STOP: 'loadster_stop_recording',
+    GET_VERSION: 'loadster_get_extension_version'
+};
+let port = null;
+
+// expose version to the webpage scope
+const script = document.createElement('script');
+script.textContent = `var loadster_extension_version = ${ browser.runtime.getManifest().version };`;
```

Comment:
Unfortunaltely there is no way to fetch version outside of extension scripts. So this is the simpliest way to expose version to a webpage.